packages/flutter_tools/test/data/intellij/mac_not_via_toolbox/Contents/Info.plist

```diff
@@ -1,38 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The original `intellij plugins path checking on mac` tests thought they were using these two Info.plists, but they actually weren't...
Unused, delete.